src/CSET/operators/plot.py

```diff
@@ -243,6 +243,24 @@ def _plot_and_save_contour_plot(
     # Add title.
     axes.set_title(title, fontsize=16)
 
+    # Add watermark with min/max/mean. Currently not user toggable.
+    axes.annotate(
+        "Min:"
+        + str(round(np.min(cube.data), 1))
+        + " Max:"
+        + str(round(np.max(cube.data), 1))
+        + " Mean:"
+        + str(round(np.mean(cube.data), 1)),
```

Comment:
```suggestion
        f"Min: {np.min(cube.data):g} Max: {np.max(cube.data):g} Mean: {np.mean(cube.data):g}"
```
f-strings are both more performant and more readable than concatenating strings.

`<number>:g` formats as a normal number while $10^{-5} < n < 10^6$, and in scientific notation otherwise, rounding to 6 significant figures.